spec/features/user/user_can_remove_items_from_cart_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.feature 'user removes items from cart' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.